docs/openapi-backend.yaml

```diff
@@ -0,0 +1,393 @@
+openapi: 3.0.3
+info:
+  title: IO Sign - Backend API
+  version: 0.0.1
+servers:
+  - url: https://app-backend.io.italia.it/api/v1/fci
```

Comment:
why fci and not 'sign'?

Reply:
Because app side "firma con Io" has always been called fci.

@hevelius

Reply:
On the app side from the beginning `fci` was used to indicate the feat flag, file name convention and other references. It's not a problem to change this url to `sign` if you want maybe can make sense because it refers to io-sign be.

Reply:
yes, @hevelius please change it to `sign` (or `io-sign` when `sign` is ambiguous)

the BE endpoint should be `/api/v1/sign/`